shim/build.rs

```diff
@@ -0,0 +1,6 @@
+fn main() {
+    println!("cargo:rustc-link-search=../cli");
```

Comment:
Once this is merged, I have some additions to make to this file to handle release vs debug. This is fine for now though, for local development